js/src/components/RelatedKeyphrasesModal.js

```diff
@@ -114,7 +116,9 @@ RelatedKeyphrasesModal.propTypes = {
 	] ),
 	currentDatabase: PropTypes.string,
 	onOpen: PropTypes.func.isRequired,
+	onOpenWitnNoKeyphrase: PropTypes.func.isRequired,
 	onClose: PropTypes.func.isRequired,
+	isLoggedInToSEMrush: PropTypes.string,
```

Comment:
Why `string` instead of `bool`?

Reply:
The only thing I've seen so far that can be used to determine whether a user is logged in is the `SEMrushRequest.OAuthToken` in the state, which is a string. However, this part is something I tried at some point and it's really not needed for this PR. Will remove for now.

Reply:
:+1: